tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/AspNetMvcIntegration.cs

```diff
@@ -30,6 +28,7 @@ public static class AspNetMvcIntegration
         internal const string HttpContextKey = "__Datadog.Trace.ClrProfiler.Integrations.AspNetMvcIntegration";
 
         private const string OperationName = "aspnet-mvc.request";
+        private const string ChildActionOperationName = "aspnet-mvc.request.child-action";
```

Comment:
Does not appear to be used.

Reply:
Good catch, question is whether:
* It should be removed as unused
* We should use a different operation name for child actions
I'm pretty sure we discussed it, but can't remember the conclusion 😬

Reply:
Thanks for catching that Robert. And indeed, we had dedided to use it Andrew.
Fixed in 9c0cc8d8d085f9af9c8fdfad1eda02b0c62390a2